### DISCUSSION TITLE
Issue with Student Verification Using School Email

### DISCUSSION THREAD

**Discussion by Tpdus911** [community/community#141043] [GitHub Education]
### Hi everyone,

Hello,

I am currently trying to complete the student verification process, but I’ve encountered an issue. My university uses “kings.ac.kr” for official school certification, but students are provided with email addresses under “email.kings.ac.kr” for security purposes. As a result, I am unable to complete the verification using my student email.

Could you please advise on how to resolve this issue? Is there an alternative way to verify my student status, or can you update the verification system to accommodate this difference?

Thank you for your assistance.

Best regards,
Seyeon

**Comment by hataiit9x** [comment]
You need to use an email address

**Reply by Tpdus911** [reply]
Our university has a contract with GitHub for Copilot services, and student verification is tied to email addresses ending with "@kings.ac.kr."
However, all students are assigned email addresses in the format of "@email.kings.ac.kr," which differs from the format used for school certification purposes. As a result, we are unable to successfully verify our student status for GitHub Copilot.
Could you please advise on how we can resolve this issue or allow for verification using our actual student email addresses?

**Reply by ThiagoAlexsander** [reply]
Contact the university support. I did that to get an email that matched the criteria.
